src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.scss

```diff
@@ -1,4 +1,6 @@
 .PosterPageSettingsControl {
+  position: relative;
```

Comment:
Why do we need `position: relative` here? We don't seem to be moving it about/repositioning it at all? Is it required?

Reply:
Removed in https://github.com/sparkletown/sparkle/pull/1299/commits/ce562c6a4616118ea904bb8f088ca751787a5f95